Rakefile

```diff
@@ -1,8 +1,18 @@
 # frozen_string_literal: true
 
 require "bundler/gem_tasks"
-require "rspec/core/rake_task"
 
+require 'github_changelog_generator/task'
+GitHubChangelogGenerator::RakeTask.new :changelog do |config|
+  config.user = 'solidusio'
+  config.project = 'solidus_dev_support'
```

Comment:
I don't get this. Is this supposed to work on extensions as well or only for this gem?

Reply:
No, this is just for the gem, I'll add a default config for the extensions in a different PR 👍 

Along with that I'm editing #121 to make it clear (to me! 😅) that it's referring to extensions and not to the gem itself 😬